lightning/src/routing/router.rs

```diff
@@ -3439,4 +3439,35 @@ mod benches {
 			idx += 1;
 		});
 	}
+
+	#[bench]
+	fn generate_mpp_routes(bench: &mut Bencher) {
+		let mut d = File::open("net_graph-2021-02-12.bin").expect("Please fetch https://bitcoin.ninja/ldk-net_graph-879e309c128-2020-02-12.bin and place it at lightning/net_graph-2021-02-12.bin");
```

Comment:
I noticed that `generate_routes` has been failing on continuous integration. Does this need to be setup there?

Reply:
Yea, I was braindead when MPP got merged - a last-minute change caused the benchmark to fail due to a bug. It should be fixed in #815, but I'm still working on finishing it. I'm gonna move bindings out of the main repo today so that we get PRs to be all-green normally, making it harder to miss a red X.